src/components/molecules/RadioGroupField/RadioGroupField.tsx

```diff
@@ -13,7 +13,7 @@ const RadioWrapper = styled.div`
 
 type RadioGroupFieldItem = {
   value: string;
-  label?: string | React.ReactNode;
```

Comment:
[Question] How come `string` was removed?

Reply:
This was some cleanup from a previous comment https://github.com/zopaUK/react-components/pull/564#discussion_r458651841

Reply:
`React.ReactNode` already includes `string`